index.html

```diff
@@ -0,0 +1,234 @@
+    <section class="about">
+        <div class="about-me">
+            <h2>About me</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <button>Get My Resume</button>
+          </div>
+          <hr>
+          <div class="skills-container">
+
+            <div class="skills-title">
+              <svg width="27" height="26" viewBox="0 0 27 26" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <rect x="0.772461" y="12.7279" width="18" height="18" transform="rotate(-45 0.772461 12.7279)" fill="#FF6B00"/>
+              </svg>
+              <h2>Language</h2>
+            </div>
+            <ul>
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Css</li>
+            </ul>
+            
+            <div class="skills-title">
+              <svg width="18" height="19" viewBox="0 0 18 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <rect y="0.455811" width="1f" height="18" fill="#FF6B00"/>
+              </svg>    
+              <h2>Frameworks</h2>
+            </div>
+            
+            <ul>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
+            
+            <div class="skills-title">
+              <svg width="18" height="19" viewBox="0 0 18 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <rect y="0.455811" width="18" height="18" rx="9" fill="#FF6B00"/>
+              </svg>     
+              <h2>Skills</h2>
+            </div>
+            <ul>
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>         
+ 
+
+    </section>
+    <!-- contact-form section. -->
+    <section class="contact-section">
+            <p class="contact-head">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            <form class="form" action="https://formspree.io/f/mvonaqyb" method="POST">
+              <div>
+                <input class="form-input" type="text" name="name" placeholder="First name" maxlength="30" required>
+                <input class="form-input" type="text" name="name" placeholder="Last name" maxlength="30" required>
+              </div>
+                <input class="form-input" type="email" name="email" placeholder="Email address" maxlength="30" required>
+                <textarea name="mail-message" id="" cols="20" rows="5" placeholder="Enter text here" maxlength="500" required></textarea>
+                <input class="send" type="submit" value="Get in touch">
+            </form>
```

Comment:
- [ ] Kindly 😊 try here to make the inputs like the Figma Designas you see here in the Figma Design there is a border below the input, so please try to make this border bottom like the Figma Design 😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/222163539-f7a55dcf-b5a8-43b9-bbe5-7b89789523b7.png)    |   ![image](https://user-images.githubusercontent.com/83360636/222163465-5383902d-34c7-4323-bee6-7b0faf1d26ee.png)  |